### PR TITLE
mirror: T3297: Optimize irrelevant error stack hints

### PR DESCRIPTION
**Pull request by jack9603301**
## Change Summary

Optimize irrelevant error stack hints

## Types of changes

 
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

https://phabricator.vyos.net/T3297

## Component(s) name

mirror

## Proposed changes

When setting the interface, the rule of port image will always be cleared.Because the TC command will return an error, when the actual error occurs,the error stack will display the execution information of
the last error command. This is an error.Now, we force the return value of these two commands to be overridden to ensure that they always return 0

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: python/vyos/ifconfig/interface.py [hunk 1]
```diff
@@ -1015,9 +1015,11 @@ def set_mirror(self):
             source_if = next(iter(self._config['is_mirror_intf']))
             config = self._config['is_mirror_intf'][source_if].get('mirror', None)
 
+        # Please don't clear 'set $?=0'. It will return the value when executing error
```
Comment (c-po) [inline python/vyos/ifconfig/interface.py]: I do not understand this comment, from my understanding this forces the return code to be 0.

**Review by c-po — CHANGES_REQUESTED** [review]

**Comment by c-po** [comment]
The redirect of stderr mimics the old implementation: https://github.com/vyos/vyatta-cfg-qos/blob/crux/scripts/vyatta-qos.pl#L332 so this is okay.

**Comment by jack9603301** [comment]
@c-po I actually used the set command, which always executed successfully, overriding the return value

File: python/vyos/ifconfig/interface.py (hunk 1 above)
Comment (jack9603301) [reply python/vyos/ifconfig/interface.py]: To simulate its normal return, one is no stderr, and two is its return value 0

Comment (jack9603301) [reply python/vyos/ifconfig/interface.py]: In fact, any set command can only result in a return value of 0, but I write this sentence just to make sense

**Pull request merged**